<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DADialogCreatePivotTable</class>
 <widget class="QDialog" name="DADialogCreatePivotTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pivot Table Guide</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxDADialogDataFrameInterpolate">
     <property name="title">
      <string>Pivot Table</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QTableView" name="tableViewParameter"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelAggfunc">
        <property name="text">
         <string>Aggregate function</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxAggfunc"/>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBoxMargins">
        <property name="toolTip">
         <string>Special All columns and rows will be added with partial group aggregates across the categories on the rows and columns</string>
        </property>
        <property name="text">
         <string>Margins</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxSort">
        <property name="toolTip">
         <string>Specifies if the result should be sorted.</string>
        </property>
        <property name="text">
         <string>Sort</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditMarginsName">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Name of the row / column that will contain the totals</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>All</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="../icon.qrc">
         <normaloff>:/DAGui/icon/accept.svg</normaloff>:/DAGui/icon/accept.svg</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../icon.qrc">
         <normaloff>:/DAGui/icon/cancel.svg</normaloff>:/DAGui/icon/cancel.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonOK</sender>
   <signal>clicked()</signal>
   <receiver>DADialogCreatePivotTable</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>DADialogCreatePivotTable</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>706</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxMargins</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditMarginsName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
